shuup/core/models/_suppliers.py

```diff
@@ -159,7 +165,34 @@ def save(self, *args, **kwargs):
             self.slug = slugify(self.name)
         return super(Supplier, self).save(*args, **kwargs)
 
-    def get_orderability_errors(self, shop_product, quantity, customer):
+    def get_supplier_module_for_product_ids(self, product_ids):
```

Comment:
this method shouldn't exist, it will make things WAY SLOWER. The supplier method should already ignore the products ids received that are not handled